### ISSUE TITLE
Data Source "mso_user" returns "invalid character '<' looking for beginning of value"

### ISSUE THREAD

**Issue by adealdag** [CiscoDevNet/terraform-provider-mso#47]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v1.0.5
on darwin_amd64
+ provider registry.terraform.io/ciscodevnet/aci v0.7.1
+ provider registry.terraform.io/ciscodevnet/mso v0.2.1

### MSO version

Version: 3.4(1a) running on top of ND Version 2.0(2h)

### APIC version and APIC Platform for Site Level Resources

* V 5.2.2f, APIC on-prem

### Affected Resource(s)

* mso_user (data source)

### Terraform Configuration Files

```hcl
terraform {
  required_providers {
    aci = {
      source  = "CiscoDevNet/aci"
      version = "0.7.1"
    }
    mso = {
      source  = "CiscoDevNet/mso"
      version = "0.2.1"
    }
  }
}

provider "aci" {
  alias = "site1"

  username = var.aci_username
  password = var.aci_password
  url      = var.aci_url_site1
  insecure = true
}

provider "aci" {
  alias = "site2"

  username = var.aci_username
  password = var.aci_password
  url      = var.aci_url_site2
  insecure = true
}

provider "mso" {
  username = var.mso_username
  password = var.mso_password
  url      = var.mso_url
  insecure = true
  platform = "nd"
}

# NDO Users
data "mso_user" "current_user" {
  username = "terraform"
}

# Create Tenant
resource "mso_tenant" "demo" {
  name         = "ms_prod_tn"
  display_name = "ms_prod_tn"

  site_associations {
    site_id = "611d411b13ea5556d1cd795b"
  }

  site_associations {
    site_id = "611d4fe213ea5556d1cd795c"
  }

  user_associations {
    user_id = data.mso_user.current_user.id
  }
}
```

### Debug Output

```
$ terraform plan
╷
│ Error: invalid character '<' looking for beginning of value
│ 
│   with data.mso_user.current_user,
│   on data_sources.tf line 11, in data "mso_user" "current_user":
│   11: data "mso_user" "current_user" {
│ 
╵
```

[Trace Logs](https://gist.github.com/adealdag/15b6d4b3b7eb428cb15cb26974c2321c)

### Expected Behavior

mso_user data source returns user information, including the id which is needed for Tenant user association

### Actual Behavior

mso_user returns an error

`Error: invalid character '<' looking for beginning of value`

### Steps to Reproduce

terraform plan or apply using the code above

1. `terraform apply`

### Important Factoids

Running NDO on top of vND

### References

* #0000

**Comment by lhercot** [comment]
Looks like duplicate of #27

**Comment by anvitha-jain** [comment]
This issue is resolved, and the changes are in #52. This will be soon be available in the next release.
